README.md

```diff
@@ -22,4 +22,4 @@ A .Net dynamic fake framework for creating all types of fake objects, mocks, stu
 * [Package](https://nuget.org/packages/FakeItEasy "FakeItEasy on NuGet")
 
 -
-FakeItEasy logo designed by Vanja Pakaski.
+FakeItEasy logo designed by [@vanpak](https://github.com/vanpak).

```

Comment:
I didn't get round to asking her, but I agree that keeping the name is better :smile: 

I've amended.